.github/workflows/wheels.yml

```
@@ -0,0 +1,101 @@
+name: Wheels Builder and Publisher
+on:
+  pull_request:
+    branches:
+      - main
+    types:
+      - closed
+
+jobs:
+  build_wheels:
+    name: Build wheels for Python SDK on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.head.ref, 'sdk-core/')
+    strategy:
+      fail-fast: false
+      matrix:
+        # macOS 13 is an Intel runner and macOS 14 is an Apple Silicon runner
+        os: [ubuntu-22.04, ubuntu-22.04-arm, windows-latest, macos-13, macos-14]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Upgrade build dependencies
+        run: python -m pip install --upgrade pip setuptools wheel
+
+
+      # Need to grab the SDK version for the wheel name
+      - name: Extract SDK Version
```

`[nit]`

You can remove one extra line:
```suggestion
        run: python -m pip install --upgrade pip setuptools wheel

      # Need to grab the SDK version for the wheel name
      - name: Extract SDK Version
```